docs/Variables.adoc

```diff
@@ -30,13 +33,21 @@ Required variable, but may be set to `none` where not applicable.
 | Name
 | Description
 
+| `pass:[__meta__]`
+| Metadata used by the Babylon infrastructure when running AgnosticD.
```

Comment:
I'd be more specific here.

Let's explain exactly what it is. The relation is not obvious. agnosticd can be run from laptop and in that case, we shouldn't care about this special variable.
I don't think this one should be considered as a core variable. It's a Babylon variable, and it is used for callbacks.

ex: it is a variable passed to agnosticd by the Babylon infrastructure. It contains information about the context etc etc.
Maybe give an example of a place it is used?

Reply:
I wish we weren't using `__meta__` at all in agnosticd. I don't think it is appropriate for anything in agnosticd to reference any metadata settings. It violates the whole concept of it being metadata. Because this is used for babylon really all I want to say here is don't use it in agnosticd.

Reply:
Ok, i agree. It wasn't clear enough since it's listed as "core variables"